File: solutions/2507-number-of-common-factors/solution.java
class Solution {
    public int commonFactors(int a, int b) {
        int n=0,m;
        if(a<b)
            m=a;
        else
            m=b;
        for(int i=1;i<=m;i++)
            if(a%i==0 && b%i==0)
                n++;
        return n;
    }
}

File: solutions/2507-number-of-common-factors/solution.js
/**
 * @param {number} a
 * @param {number} b
 * @return {number}
 */
var commonFactors = function(a, b) {
    var n=0;
        for(let i=1;i<=b;i++)
            if(a%i==0 && b%i==0)
                n++;
        return n;
};
